app/qml/map/MMRecordingTools.qml

```diff
@@ -314,6 +315,19 @@ Item {
     }
   }
 
+  Component.onCompleted: {
+    if ( root.centerToGPSOnStartup ) {
+      // center to GPS
+      if ( gpsStateGroup.state === "unavailable" ) {
+          __notificationModel.addError( "GPS currently unavailable." );
+          return;
+      }
+
+      root.map.mapSettings.centeredToGPS = true;
```

Comment:
This looks strange, there's no `InputMapSettings::centeredToGPS`. Did you mean `mapTool.centeredToGPS = true`?